src/gurt/hash.c

```diff
@@ -875,8 +875,10 @@ d_hash_table_create(uint32_t feats, uint32_t bits, void *priv,
 		D_GOTO(out, rc = -DER_NOMEM);
 
 	rc = d_hash_table_create_inplace(feats, bits, priv, hops, htable);
-	if (rc)
+	if (rc) {
 		D_FREE(htable);
```

Comment:
D_FREE internally do the "(ptr) = NULL;", so probably need not explicitly zero it again.

Reply:
You are right..probably there are other memory corruptions cause my problem. thanks.